app/lib/submission_builder/ty2024/states/id/id_return_xml.rb

```diff
@@ -32,11 +32,25 @@ def supported_documents
             supported_docs = [
               {
                 xml: SubmissionBuilder::Ty2024::States::Id::Documents::Id40,
-                pdf: nil,
+                pdf:  PdfFiller::Id40Pdf,
                 include: true
               },
+              {
+                xml: nil, # TODO 39RForm xml to be added in future story
+                pdf:  PdfFiller::Id39rPdf,
+                include: @submission.data_source.dependents.count > 4
+              },
             ]
-            
+            # If there are more than 7 dependents total (the State XML field can handle a max of 20), another copy of the ID Form 39R PDF is used to handle these cases
```

Comment:
Sorry I'm confused about this so just going to make sure I understand correctly. The first 4 are on the Id40pdf, the next three are on the id39r and then an additional 13 can be added in set of 3 using the Id39r-additional pdf right?

Reply:
yes!